src/xcode/ENA/ENA/Source/Models/Exposure/__tests__/ExposureManagerTests.swift

```diff
@@ -7,6 +7,21 @@ import ExposureNotification
 @testable import ENA
 
 class ExposureManagerTests: CWATestCase {
+	
+	func test_requesting_userPermission_in_EOL() {
```

Comment:
```suggestion
	func testRequestEnableUserPermission_EOL_noError() {
```

or something in this direction ...